GeeksCoreLibrary/Modules/Templates/Services/PagesService.cs

```diff
@@ -215,22 +233,26 @@ public async Task<PageViewModel> CreatePageViewModelAsync(List<string> externalC
                 foreach (var template in templates)
                 {
                     externalCss.AddRange(template.ExternalFiles);
-                    switch (template.InsertMode)
+
+                    if (!String.IsNullOrWhiteSpace(template.Content))
```

Comment:
Kun je deze `if` omdraaien, zodat de code minder hoeft in te springen en dit consistent is met de rest van de GCL?
```c#
if (String.IsNullOrWhiteSpace(template.Content))
{
    continue;
}
```

Reply:
De `if` is omgedraaid.